lua/cps.lua

```diff
@@ -5,7 +5,6 @@ return function (net_conf, gk_conf, gt_conf, lls_conf, numa_table)
 	--
 
 	-- These parameters should likely be initially changed.
-	local kni_kmod_path = "/home/user/gatekeeper/dependencies/dpdk/build/kmod/rte_kni.ko"
```

Comment:
Update the documentation on the wiki and use `"/home/user/gatekeeper/dependencies/dpdk/build/kmod/rte_kni.ko"` as an example. Also mention when this parameter is meant to be used (see #211 for this).